cms/templates/program_authoring.html

```diff
@@ -1,9 +1,11 @@
+<%namespace name='static' file='static_content.html'/>
 <%! from django.utils.translation import ugettext as _ %>
 
 <%inherit file="base.html" />
 <%block name="title">${_("Program Administration")}</%block>
 
 <%block name="header_extras">
+<%static:css group='style-edx-icons'/>
```

Comment:
Oh, disregard my note about this causing performance issues across Studio - you've scoped this to your programs-based views/template. Thanks!